Kitodo/src/main/java/de/sub/goobi/forms/HelperForm.java

```diff
@@ -47,83 +42,13 @@
 public class HelperForm implements Serializable {
     private static final long serialVersionUID = -5872893771807845586L;
     private transient ServiceManager serviceManager = new ServiceManager();
-    private static final Logger logger = LogManager.getLogger(HelperForm.class);
-    static final String MAIN_JSF_PATH = "/pages";
-    private static final String IMAGE_PATH = "/pages/images";
     private static final String CSS_BASE_PATH = "/WEB-INF/resources/css";
     private static final String CSS_PATH = "/userStyles";
 
     public String getVersion() {
         return GoobiVersion.getBuildversion();
     }
 
-    /**
-     * Get application logo.
-     *
-     * @return link to logo
-     */
-    public String getApplicationLogo() {
-        String logo = getServletPathWithHostAsUrl() + IMAGE_PATH + "/template/";
-        logo += ConfigCore.getParameter("ApplicationLogo", "kitodo-header-logo.svg");
-        return logo;
-    }
-
-    /**
-     * Get application header background.
-     *
-     * @return css style for header
-     */
-    public String getApplicationHeaderBackground() {
-        String headerImage = getServletPathWithHostAsUrl() + IMAGE_PATH + "/template/";
-        headerImage += ConfigCore.getParameter("ApplicationHeaderBackground", "goobi_meta_verlauf.jpg");
-        /* wenn ein Background angegeben wurde, dann diesen jetzt strecken */
-        if (headerImage.length() > 0) {
-            headerImage = "background: url(" + headerImage + ") repeat-x;";
-        }
-        return headerImage;
-    }
-
-    // TODO: Change the defaults
-    public String getApplicationHeaderTitle() {
-        return ConfigCore.getParameter("ApplicationHeaderTitle", "Goobi - Universitätsbibliothek Göttingen");
-    }
-
-    public String getApplicationTitle() {
-        return ConfigCore.getParameter("ApplicationTitle", "http://goobi.gdz.uni-goettingen.de");
-    }
-
-    public String getApplicationTitleStyle() {
-        return ConfigCore.getParameter("ApplicationTitleStyle", "font-size:17; font-family:verdana; color: black;");
-    }
-
-    public String getApplicationWebsiteUrl() {
-        return getServletPathAsUrl();
-    }
-
-    public String getApplicationWebsiteMsg() {
-        String result = ConfigCore.getParameter("ApplicationWebsiteMsg", getApplicationWebsiteUrl());
-        return Helper.getTranslation(result);
-    }
-
-    public String getApplicationHomepageMsg() {
-        String result = ConfigCore.getParameter("ApplicationHomepageMsg", getApplicationWebsiteUrl());
```

Comment:
You should even remove `ApplicationHomepageMsg`  entry including the describing comment from all existing `kitodo_config.properties` files.